homeassistant/components/advantage_air/switch.py

```diff
@@ -33,7 +37,7 @@ class AdvantageAirFreshAir(AdvantageAirEntity, SwitchEntity):
 
     _attr_icon = "mdi:air-filter"
 
-    def __init__(self, instance, ac_key):
+    def __init__(self, instance: advantage_air, ac_key: int) -> None:
```

Comment:
ac_key is a string
https://github.com/home-assistant/core/blob/dev/tests/components/advantage_air/fixtures/getSystemData.json#L3